### PR TITLE
Trigger boolean nimble operators

### PR DESCRIPTION
**Pull request by Igor-Palaguta**
PR for https://github.com/realm/SwiftLint/issues/2613

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -32,6 +32,10 @@
 * Make `redundant_objc_attribute` rule autocorrectable.  
   [Daniel Metzing](https://github.com/dirtydanee)
 
+* `nimble_operator` now warns about `beTrue()` and `beFalse()`
```
Comment (marcelofabri) [inline CHANGELOG.md]: requires two trailing spaces after a period as described in CONTRIBUTING.md: https://github.com/realm/SwiftLint/blob/master/CONTRIBUTING.md#tracking-changes

**Comment by marcelofabri** [comment]
@Igor-Palaguta the trailing spaces should be after the line describing the change (before the line with your name), not on that empty line

**Comment by Igor-Palaguta** [comment]
I hope now it is correct 😄

**Comment by marcelofabri** [comment]
Thanks @Igor-Palaguta 🎉

**Pull request merged**